Sources/JavaKitConfigurationShared/GenerationMode.swift

```diff
@@ -76,3 +76,29 @@ extension JExtractMinimumAccessLevelMode {
     .public
   }
 }
+
+
+/// Configures how memory should be managed by the user
+public enum JExtractMemoryManagementMode: String, Codable {
+  /// Force users to provide an explicit `SwiftArena` to all calls that require them.
+  case forceExplicit
+
+  /// Provide both explicit `SwiftArena` support
+  /// and a default global automatic `SwiftArena` that will deallocate memory when the GC decides to.
+  case allowAutomatic
+
+  /// Force all memory management to a default global automatic `SwiftArena`
+  /// that will deallocate memory when the GC decides to.
+  case forceAutomatic
```

Comment:
I'd be leaning towards not having this option -- you can do allow global and just not pass the arena anytime. It's not quite worth preventing people from doing explicvit management which is usually the "right thing" IMHO

Reply:
fine by me